### DISCUSSION TITLE
Periodic access (upload, admin) issues

### DISCUSSION THREAD

**Discussion by Th3L1nuxGuy** [requarks/wiki#6287] [Help / Questions] [answered]
Wiki.js 2.5.297 -- running as systemd service
Apache 2.4.53 -- reverse proxy to wiki.js
Authentication: LDAP / Active Directory
CentOS 9 Stream

I'm trying to get a new Wiki.js instance up and I'm running into a peculiar problem -- one I'm hoping is just something I'm doing wrong. While editing content on the wiki, there are occasions where images suddenly don't upload to the wiki (I get 403 forbidden errors from wiki.js). At the very same time, my "gear" icon in the upper-right corner (linking to the admin page) disappears. The nearby user menu (clicking on the profile icon) doesn't show a logout or profile option in the menu. It's like I've been forcibly logged out, except I can still edit posts (I just can't upload new files). 

Sometimes navigating back to the sign-in page and signing back in fixes it. Sometimes I have to remove browser cookies and then sign back in. Once I'm back in, I can upload images, get to the admin page, etc. But after a period of time -- sometimes 30 minutes, sometimes hours later -- I'm dealing with the same problem again.

I've been playing with token expiration and token renewal period settings, trying to see if anything fixes it. Increasing the token renewal period settings seemed to help some.

Any thoughts on what I could do to help this? Help is certainly appreciated!

**Comment by pgi-jsanchez** [comment, marked as answer]
A workarround that you can do, it's to navigate to `<site>/logout`, that will force to remove the cookie and log out, being able to relogin again. 

It's a issue that I currently have and idk if there's another disussion or issue involve into this, but at least I give you this tip meanwhile a solution comes. I hope as well that it's something that I did wrong during the configuration process or something like that.

**Reply by Th3L1nuxGuy** [reply]
This isn't the best solution, but it is definitely a reasonable workaround. Here's hoping the issue is squared away in version 3, which is looking pretty impressive at this point.

Thanks for the info!

**Reply by gymall** [reply]
Hello,
i got the same error. Do you find out a fix?
Best regards

**Reply by Th3L1nuxGuy** [reply]
I did not. I ended up removing the reverse proxy with Apache. Maybe one day I'll revisit the issue, but for now I'm just running wiki.js directly on port 443.
